docs/howto/providers/azure.rst

```diff
@@ -0,0 +1,16 @@
+
+Deleting or stopping your virtual machine
+===========================================
+
+After you have finished using your LTJH you might wanto to either Stop or completely delete the Virtual Machine to avoid incurring in subsequent costs. 
+The difference between these two approaches is that **Stop** will keep the VM resources but will effectively stop any compute / runtime activities. 
+If you choose to delete the VM then all the resources associated with it will be wiped out.
```

Comment:
If you stop the VM, are you still charged for the disk? If so we should mention it.